src/renderers/dom/shared/wrappers/__tests__/ReactDOMSelect-test.js

```diff
@@ -497,9 +497,7 @@ describe('ReactDOMSelect', () => {
     stub = ReactDOM.render(stub, container);
     var node = ReactDOM.findDOMNode(stub);
 
-    expect(() => ReactTestUtils.Simulate.change(node)).not.toThrow(
-      "Cannot set property 'pendingUpdate' of null"
-    );
+    expect(() => ReactTestUtils.Simulate.change(node)).not.toThrow();
```

Comment:
We should never test for absence of a particular error message since that might cover up errors when messages change.